### PR TITLE
Fix namespaces

### PR DESCRIPTION
**Pull request by southeo**
Few general fixes:
- Fix formatting according to DiSSCo practice
- fix missing namespace for "type"
- Add oa namespace to selector (should this be ods?)
- Capitalized constants "FieldValueSelector" and "ClassValueSelector"

### CODE REVIEW

File: data-model/annotations/schema/annotation.json [hunk 1]
```diff
@@ -2,99 +2,207 @@
...
+        "ods:type": {
+          "type": "string"
+        },
+        "oa:selector": {
           "type": "object",
           "oneOf": [
             {
               "type": "object",
               "properties": {
-                "type": { "const": "fieldValueSelector" },
```
Comment (sharifX) [inline data-model/annotations/schema/annotation.json:24]: this should be FieldValueSelector

File: data-model/annotations/schema/annotation.json [hunk 2]
```diff
@@ -2,99 +2,207 @@
...
-              "required": ["ods:type", "ods:field"]
+              "required": [
+                "ods:type",
+                "ods:field"
+              ]
             },
             {
               "type": "object",
               "properties": {
-                "type": { "const": "classValueSelector" },
```
Comment (sharifX) [inline data-model/annotations/schema/annotation.json:32]: This should be ClassValueSelector

File: data-model/annotations/schema/annotation.json [hunk 3]
```diff
@@ -2,99 +2,207 @@
...
-        "ods:id": { "type": "string" },
-        "ods:type": { "type": "string" },
-        "selector": {
+        "ods:id": {
+          "type": "string"
+        },
+        "ods:type": {
+          "type": "string"
+        },
+        "oa:selector": {
```
Comment (sharifX) [inline data-model/annotations/schema/annotation.json:34]: this is correct. selector is from oa namespace https://www.w3.org/ns/anno.jsonld

File: data-model/annotations/schema/annotation.json [hunk 4]
```diff
@@ -2,99 +2,207 @@
   "$schema": "http://json-schema.org/draft-2020-12/schema#",
   "type": "object",
   "properties": {
-    "ods:id": { "type": "string" },
-    "ods:type": { "const": "Annotation" },
```
Comment (sharifX) [inline data-model/annotations/schema/annotation.json:6]: this "type" is rdf:type based on w3c https://www.w3.org/ns/anno.jsonld. The other "types" are FDO Types so should ods.

**Review by sharifX — APPROVED** [review]

File: data-model/annotations/schema/annotation.json [hunk 5]
```diff
@@ -2,99 +2,207 @@
...
-    "version": { "type": "integer" },
+    "ods:id": {
+      "type": "string"
+    },
+    "ods:type": {
+      "const": "Annotation"
+    },
+    "version": {
+      "type": "integer"
+    },
```
Comment (TomDijkema) [inline data-model/annotations/schema/annotation.json:13]: I think @samleeflang mentioned during the planning that version can have 'ods' prefix

File: data-model/annotations/schema/annotation.json (hunk 4 above)
Comment (southeo) [reply data-model/annotations/schema/annotation.json:6]: so oa:type?

Comment (sharifX) [reply data-model/annotations/schema/annotation.json:6]: No. rdf:type. anno.json refers to the rdf name space in the json ld context. https://www.w3.org/1999/02/22-rdf-syntax-ns#

**Review by TomDijkema — APPROVED** [review]

**Pull request merged**